### DISCUSSION TITLE
Images for entries

### DISCUSSION THREAD

**Discussion by parfait8566** [Jitendex/Jitendex#25] [Ideas]
Images are not currently supported as far as I know, but I'll still leave them here to see what you think

**Comment by parfait8566** [comment]
ゴロゴロ (sense 2)
https://stampo.fun/s0003_20/ 
Seems free to use: https://stampo.fun/license/

**Reply by stephenmk** [reply]
I'm interested in images that are [publicly licensed](https://en.wikipedia.org/wiki/Public_copyright_license#Types_of_copyright_license). The conditions for using the images on that website are quite restrictive in comparison.

Dictionaries such as daijirin, daijisen, sanseido kokugo jiten, and shinsen kokugo jiten all contain images in some entries. When deciding which entries in Jitendex should have images, I'll probably use those references as a guide.

It may be a while before I add image support to Jitendex. For the foreseeable near future, I don't have time to add new features.
